docs/contributing/checklist.md

```diff
@@ -0,0 +1,167 @@
+---
+id: checklist
+title: Contribution Checklist
+---
+
+This document includes a checklist that summarizes the list of files that you need in order to contribute to the Cortex XSOAR content repository. Please make sure you have everything before you open a Pull Request (unless stated otherwise).
+
+Keep in mind that content packs can contain multiple types of entities, such as Integrations, Automations, Playbooks, Incident Types, Incident Fields, and so on.
+
+Depending on whether your content pack is aiming to be *certified* or not, there will be different requirements.
```

Comment:
```suggestion
Depending on whether your content pack is aiming to be [certified](https://xsoar.pan.dev/docs/partners/certification) or not, there will be different requirements.
```

Reply:
@fvigo Why not referencing the certification document?

Reply:
sorry missed, this it's ok now